### DISCUSSION TITLE
Nextjs inbuilt Images are overlapping?

### DISCUSSION THREAD

**Discussion by 18121A05L2** [vercel/next.js#39972] [Help] [answered]
hey can anyone help me with this ?

https://stackoverflow.com/questions/73499353/nextjs-inbuilt-images-are-overlapping

**Comment by icyJoseph** [comment, marked as answer]
I checkout your app, and for the most part, I only see that the issue seems to be the `z-100`, which is not a valid Tailwind z-index. Change it to `z-50`.

In general a few things:

- Everything seems to be a div, use other tags
- Use `isolate` for the header and for the rest of the app content, to create two stacking contexts at that level

**Reply by 18121A05L2** [reply]
thanks bro , its working fine now

**Reply by icyJoseph** [reply]
This tool can help out when you are dealing with stacking contexts, and z-indexes, https://chrome.google.com/webstore/detail/css-stacking-context-insp/apjeljpachdcjkgnamgppgfkmddadcki.

**Reply by 18121A05L2** [reply]
chome extension is very much useful bro thanks 

bro how to isolate the header , i am actually hearing this word now only and what are the best other tags helpful rather than div , i know div and section only

**Reply by icyJoseph** [reply]
There's a CSS property called `isolation`, https://developer.mozilla.org/en-US/docs/Web/CSS/isolation, in Tailwind you can do `isolate` as a classname.

`div`, `section`, `article`, it depends on the semantics of course, but from the quick look I took, it was mostly just divs, perhaps that was a hasty conclusion, but it is good to be mindful of HTML semantics.

**Reply by 18121A05L2** [reply]
sure bro will go through that

**Reply by akashroy1949** [reply]
@icyJoseph Can you please explain what this isolation does and how it works?
I was also going through the same issue when I used to position relative after assigning z-index it worked.
